win32ss/user/ntuser/sysparams.c

```diff
@@ -371,6 +372,15 @@ NtUserUpdatePerUserSystemParameters(
         g_PaintDesktopVersion = FALSE;
     bResult = TRUE;
 
+    DWORD dwValue = 0;
+    NTSTATUS Status = RegReadUserSetting(L"Control Panel\\Desktop",
+                                             L"WindowArrangementActive",
+                                             REG_SZ, &dwValue, sizeof(dwValue));
```

Comment:
This is wrong, `RegReadUserSetting` returns a boolean value and not a NTSTATUS one. Also there's extra space in the last two lines.

See https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/misc/registry.c;hb=4c731adc041e611622c2dbb39cd67d49bd48ed19#l158